install/installer/pkg/components/ide-service/ide-configmap.json

```diff
@@ -94,10 +94,10 @@
         "versions": [
           {
             "version": "2024.1",
-            "image": "{{.Repository}}/ide/intellij:commit-ef400309563b040b84d9588862805ce2f26d688a",
+            "image": "{{.Repository}}/ide/intellij:commit-1bc46bd2a58dbc9033313519453939d895f16fce",
             "imageLayers": [
-              "{{.Repository}}/ide/jb-backend-plugin:commit-ef400309563b040b84d9588862805ce2f26d688a",
-              "{{.Repository}}/ide/jb-launcher:commit-614cc59b70f3ebfe26b3a31a32088da54af1c9db"
+              "{{.Repository}}/ide/jb-backend-plugin:commit-1bc46bd2a58dbc9033313519453939d895f16fce",
```

Comment:
@akosyakov we don't expect them (intellij/goland) to be changed right? Once it's pinned, everything inside should be pinned unless we need to patch 

(it's changed because we fetch latest main installer's values, and backend-plugin is just updated.)

If so, I'll see if I can improve or upgrade script

Reply:
Or consider it just like a pin for latest, then we always keep it up-to-date? Like changes below

Reply:
Could it be effect on Pudong's PR to separate main and branch caches somehow? Not sure why we get new image here for intelijg?

Reply:
That's because we always get image info from latest build
https://github.com/gitpod-io/gitpod/blob/9382d33ee521e6a72d763f906b24dd53893103df/components/ide/jetbrains/image/gha-update-image/lib/pin-version.ts#L57-L58
, and update it when we find it new one https://github.com/gitpod-io/gitpod/blob/9382d33ee521e6a72d763f906b24dd53893103df/components/ide/jetbrains/image/gha-update-image/lib/pin-version.ts#L108

Reply:
Pinned versions should be added old version when we're going to upgrade to new one, then never touch it again, current version should not inside it. Otherwise we need to update it, and it's not actually pin

Reply:
How new logic will be then? If such version already pinned then do nothing? 

I think what is most important that we should guruantee that 
latest stable resolved from pinned should be the same like unpinned stable.